scripts/workflows.ini

```diff
@@ -0,0 +1,109 @@
+
+test_type = "PR tests"
+checkpoint_tests = 1
+checkpoint_examples = 1
+checkpoint_mpi = 1
+checkpoint_warnings_as_errors = 0
+checkpoint_docs = 0
+checkpoint_asan = 0
+build_type = release
+
+ulimit_core = 0
+code_coverage = 0
+repo = lifflander1/checkpoint
+arch = amd64
+is_ubuntu = 1
+distro = 18.04
+linux = ubuntu
+cache_dir = "~/.local/cache/"
+cache_name = "[% linux %]-[% compiler %]-cache"
+docker_target = "[% linux %]-cpp"
+triggers = "on:\\n  push:\\n    branches:\\n      - develop\\n      - 1.*\\n  pull_request:"
+warning = "###############################################################################\\n############## Warning this is a generated file---do not modify ###############\\n###############################################################################"
+job_name = build
+workflow_runs_on = ubuntu-latest
+build_root = "~/.local/cache/${{ env.ARCH }}-[% linux %]-${{ env.UBUNTU }}-${{ env.COMPILER }}-cache/"
+linux_env = "      UBUNTU: [% distro %]"
+
+[PR-tests-gcc-5]
+test_configuration = "gcc-5, ubuntu, mpich"
+compiler_type = gnu
+compiler = gcc-5
+output_name = .github/workflows/dockerimage-gcc-5-ubuntu-mpich.yml
+
+[PR-tests-gcc-6]
+test_configuration = "gcc-6, ubuntu, mpich"
+compiler_type = gnu
+compiler = gcc-6
+output_name = .github/workflows/dockerimage-gcc-6-ubuntu-mpich.yml
+
+[PR-tests-gcc-7]
+test_configuration = "gcc-7, ubuntu, mpich"
+compiler_type = gnu
+compiler = gcc-7
+output_name = .github/workflows/dockerimage-gcc-7-ubuntu-mpich.yml
+code_coverage = 1
+
+[PR-tests-gcc-8]
+test_configuration = "gcc-8, ubuntu, mpich, address sanitizer"
```

Comment:
build configurations possibly missing:
- mpich disabled (for "general purpose" scenario?)
- debug
- warnings as errors enabled

Reply:
Added warnings as errors build. I don't think that non-MPI makes sense because it just disables the MPI examples.